distro/installer/centos

```diff
@@ -2,7 +2,7 @@
 
 detect_system
 
-if (( $_system_version >= 8 )); then
+if (( $_system_version >= 9 )); then
```

Comment:
thanks, this logic can be kept, as it is still true even if lkp-tests doesn't support version 8 anymore.